docs/actions/assert.md

```diff
@@ -0,0 +1,235 @@
+    |:--------:|:----------------|:------------|:---------------:|:----------------------:|
+    |    *     | `actual`        | String      |                 | actual string value    |
+    |    *     | `expected`      | String      |                 | expected string value  |
+    |    *     | `mode`          | String      |                 | comparison mode        |
+
+=== "Outputs"
+    No outputs
+
+
+### Example
+=== "Kotlin"
+    ``` kotlin
+    CompareTask(
+        actual = "chutney",
+        expected = "chutney",
+        mode = "EQUALS" // case insensitive
+    ) 
+    ```
+!!! note "mode can be"
+
+    | mode                             |          desscription                  |
+    |:--------------------------------:|:---------------------------------------|
+    | `equals`                         | actual and expected are equals         |
+    | `not equals` / `not-equals`      | actual and expected are not equals     |
+    | `contains`                       | actual contains expectd                |
+    | `not-contains` / `not contains`  | actual doesn't contain expectd         |
+    | `greater-than` / `greater than`  | actual is greater than expected        |
+    | `less-than` / `less than`        | actual is less than expected           |
+
+## Json compare
+compare two json inputs(the hole content or only some nodes) using a comparison mode.
+
+=== "Inputs"
+
+    | Required | Name            | Type        |     Default      |    Description                  |
+    |:--------:|:----------------|:------------|:----------------:|:-------------------------------:|
+    |    *     | `document1`     | String      |                  | first json                      |
+    |    *     | `document2`     | String      |                  | second json                     |
+    |    *     | `comparingPaths`| Map         | Map.of("$", "$") | to be comapared nodes [json paths](https://github.com/json-path/JsonPath){:target="_blank"} |
+    |    *     | `mode`          | String      |  STRICT          | comparison mode                 |
+
+=== "Outputs"
+    No outputs
+
+
+### Example
+=== "Kotlin"
+    ``` kotlin
+    JsonCompareTask(
+        document1 = "{\"string\": \"string_value\", \"object\":{\"number\":3}, \"list\": [1, 2, 3]}",
+        document2 = "{\"string\": \"string_value\", \"object\":{\"number\":3}, \"list\": [1, 2, 3]}"
+    )
+    ```
+!!! note "mode can be"
+
+    | mode       |          desscription                                                                                    |
+    |:----------:|:---------------------------------------------------------------------------------------------------------|
+    | `STRICT`   | strict comparison                                                                                        |
+    | `LENIENT`  | even if the document1 contains extended fields (which document2 doesn't have), the test will still pass  |
+
+# Step validation
```

Comment:
I will precise this at the begining at the page

Reply:
Same, at the beginning and I would give a lot more explanations. 
if you don't know how to do right now, no worry, we can improve it later